test/checks/keyboard/has-at-least-one-main.js

```diff
@@ -0,0 +1,97 @@
+describe('has-at-least-one-main', function () {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+	var checkContext = {
+		_data: null,
+		data: function (d) {
+			this._data = d;
+		},
+	};
+
+	afterEach(function () {
+		fixture.innerHTML = '';
+		checkContext._data = null;
+	});
+
+	
+	it('should return false if main does not exist', function () {
+		var node = document.createElement('html');
```

Comment:
Was there a reason to add an HTML element here? The fixture should already exist in the document body.